Makefile

```diff
@@ -591,7 +592,7 @@ prv : ;@$(MAKE) $(MFLAGS) V=$(V) prove
 prove-all :
 	+$(MAKE) prove realsearch=%
 
-junit-t% : BACKENDS += $(TEST_BACKENDS)
+junit-t% : BACKENDS = $(BACKENDS) $(TEST_BACKENDS)
```

Comment:
I have no clue why this line would result in junit running each backend twice but this change does not.